src/main/java/com/xiaomi/infra/pegasus/tools/interceptor/TableInterceptor.java

```diff
@@ -0,0 +1,13 @@
+package com.xiaomi.infra.pegasus.tools.interceptor;
```

Comment:
Interceptor is a component belongs to RPC module. There's no interceptor if there's no RPC. So it should be placed as a single file in rpc.

```
package com.xiaomi.infra.pegasus.rpc;
```